teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
```

Comment:
- [ ] Good job According to the project requirement your constructor has a name, age, and `parent_permission `as parameters, and `name` and `parent_permission` are optional and have default values of `"Unknown"` and `true` . Please add the missing parameters and default value :+1: